src/components/Home/Notification/index.js

```diff
@@ -15,10 +18,47 @@ export default () => {
       {dark ? <Notif className="notif" /> : <NotifLight className="notif" />}
       <div className="content">
         <h1> Get Notified </h1>
-        <p> Get instant updates about GeeksBlaBla </p>
-        <div className="actions">
-          <GetNotification />
+        <p>
+          The best way to get notified about live streams is by subscribing to
```

Comment:
Simpler wording: The to get live streams notifications subscribe to our Youtube Channel and Facebook Page (Make sure to enable notifications 😉)